### ISSUE TITLE
content-type: text is added to header if content-type is not present

### ISSUE THREAD

**Issue by Gr33nbl00d** [containous/traefik#4761]
### What did you do?

configured 1 frontend and 1 backend with 1 server without any additional config

```toml
[entryPoints]
  [entryPoints.http]
  address = ":8880"
[backends]
  [backends.backend1]
    [backends.backend1.servers]
      [backends.backend1.servers.server0]
        url = "http://10.221.132.17:80"
		
[frontends]
  [frontends.frontend1]
    entryPoints = ["http", "https"]
    backend = "backend1"
    passHostHeader = true
```

requested a stylesheet stlye.css

### What did you expect to see?

Unmodified (non existing) content-type

### What did you see instead?
Traefik added a content-type: text to the header

### Output of `traefik version`: (_What version of Traefik are you using?_)
1.7.10

### Traefik generated header:
```
Accept-Ranges: bytes
Content-Length: 6717
Content-Type: text/plain; charset=utf-8
Date: Fri, 12 Apr 2019 11:53:43 GMT
Etag: W/"6717-1551984310000"
Last-Modified: Thu, 07 Mar 2019 18:45:10 GMT
Server: Apache-Coyote/1.1
```
### Original server header:
```
Accept-Ranges: bytes
Content-Length: 6717
Date: Fri, 12 Apr 2019 11:50:43 GMT
ETag: W/"6717-1551984310000"
Last-Modified: Thu, 07 Mar 2019 18:45:10 GMT
Server: Apache-Coyote/1.1
```

It was already reported in bug 
#3131

But you simply closed it with a Bot! Do you think problems go away by closing them????
I dont think i will use traefik as i seen such big issues are simply ignored...
Bad behaviour...

**Comment by geraldcroes** [comment]
Hi @Gr33nbl00d , and thank you for reporting this issue.

As you noticed, this issue you're reporting has already been reported in the past, and unfortunately, despite our best effort, we weren't able to reproduce it (as @Ullaakut said [there](https://github.com/containous/traefik/issues/3131#issuecomment-433301043))

Please excuse our bot, it's doing its best to close issues tagged with "waiting-for-feedback" that are 90 days old without activity (as documented [there](https://docs.traefik.io/v2.0/contributing/maintainers/#myrmica-bibikoffi)). But it's not a bad bot per se, it really wants to keep things open, reason why it says you should feel free to re-open the/an issue 👍 

So, in the end, we're happy to finally find someone able to provide us with a reproducible use case 🎉 Would you be so kind to send us one? (don't let the bot do its deed 😨) We'll sure fix it ASAP if it's confirmed.

[Thank You!](https://github.com/containous/traefik/blob/master/CODE_OF_CONDUCT.md)

**Comment by Gr33nbl00d** [comment]
Ok i will put an example together on monday so you are able to reproduce it

**Comment by n2ygk** [comment]
@Gr33nbl00d OK, so did you come up with the example? I think I am seeing this as well...

**Comment by n2ygk** [comment]
Here's my example in which the backend returns `Content-Type: text/css` at [this URL](http://ac45devapp01.cc.columbia.edu:9090/static/openapi/swagger-dist-ui/swagger-ui.css) but traefik returns `Content-Type: text/plain; charset=utf-8` at [this URL](https://ac45devapp01.cc.columbia.edu/static/openapi/swagger-dist-ui/swagger-ui.css).

These URLs are public, feel free to poke at them, although I may break things as I keep trying to figure this. Using a related docker-compose.yml, this works OK on localhost (MacOS Mojave/docker 18.09.2) but fails on RHEL 7.6/docker 1.13.1).

Traefik docker image is: 
```
$ docker inspect traefik
[
    {
        "Id": "sha256:fb5ce07475c690dbfbc475ebf0b57bf2e8dc5bc9a89dce64616738685e708080",
        "RepoTags": [
            "traefik:latest"
        ],
```
The docker-compose.yml contains:
```yaml
version: '3.1'

volumes:
  local_staticfiles: {}

networks:
  frontend:
  storage:
  api:
  traefik:

services:
  traefik:
    image: traefik
    command: --api --docker
    ports:
      # only expose https to outside world
      - "443:443"   # SSL
      - "8080:8080" # Traefik dashboard
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock
      - "$PWD/traefik/config/traefik.toml:/etc/traefik/traefik.toml"
      - "$PWD/traefik/certs/:/certs/"
    labels:
      - "traefik.enable=true"
      - "traefik.docker.network=traefik"
    depends_on:
      - apim_gateway
      - apim_managementui
      - apim_managementapi
    networks:
      - frontend
      - api
      - traefik

  elasticsearch:
    hostname: elasticsearch
    image: docker.elastic.co/elasticsearch/elasticsearch:5.4.3
    volumes:
      - ./data/elasticsearch:/usr/share/elasticsearch/data:Z
    environment:
      - http.host=0.0.0.0
      - transport.host=0.0.0.0
      - xpack.security.enabled=false
      - xpack.monitoring.enabled=false
      - cluster.name=elasticsearch
      - bootstrap.memory_lock=true
      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
    labels:
      - "traefik.enable=false"
    ulimits:
      memlock:
        soft: -1
        hard: -1
      nofile: 65536
    networks:
      - storage

  mongodb:
    hostname: mongodb
    image: mongo:3.4
    volumes:
      - ./mongo/docker-entrypoint-initdb.d:/docker-entrypoint-initdb.d
      - ./data/mongo:/data/db:Z
      - ./backups/mongo:/backups:Z
    labels:
      - "traefik.enable=false"
    networks:
      - storage

  apim_gateway:
    hostname: demo-gateway
    image: graviteeio/gateway:1.24.0
    environment:
      - gravitee_management_mongodb_uri=mongodb://mongodb:27017/gravitee?serverSelectionTimeoutMS=5000&connectTimeoutMS=5000&socketTimeoutMS=5000
      - gravitee_ratelimit_mongodb_uri=mongodb://mongodb:27017/gravitee?serverSelectionTimeoutMS=5000&connectTimeoutMS=5000&socketTimeoutMS=5000
      - gravitee_reporters_elasticsearch_endpoints_0=http://elasticsearch:9200
    labels:
      - "traefik.frontend.rule=Host:ac45devapp01.cc.columbia.edu;PathPrefix:/v1"
      - "traefik.docker.network=traefik"
      - "traefik.backend=apim_gateway"
      - "traefik.port=8082"
    ports:
      - "8000:8082"
    depends_on:
      - mongodb
      - elasticsearch
      - demoapp
    networks:
      - storage
      - frontend
      - api
      - traefik

  apim_managementui:
    hostname: managementui
    image: graviteeio/management-ui:1.24.0
    environment:
      - gravitee_management_mongodb_uri=mongodb://mongodb:27017/gravitee?serverSelectionTimeoutMS=5000&connectTimeoutMS=5000&socketTimeoutMS=5000
      - gravitee_ratelimit_mongodb_uri=mongodb://mongodb:27017/gravitee?serverSelectionTimeoutMS=5000&connectTimeoutMS=5000&socketTimeoutMS=5000
      - gravitee_analytics_elasticsearch_endpoints_0=http://elasticsearch:9200
      - gravitee_jwt_cookiepath=/management
      - gravitee_jwt_cookiesecure=true
      - gravitee_jwt_cookiedomain=ac45devapp01.cc.columbia.edu
      - MGMT_API_URL=https:\/\/ac45devapp01.cc.columbia.edu\/management\/
    labels:
      - "traefik.frontend.rule=Host:ac45devapp01.cc.columbia.edu;PathPrefixStrip:/portal"
      - "traefik.docker.network=traefik"
    depends_on:
      - mongodb
      - elasticsearch
    networks:
      - storage
      - frontend
      - traefik

  apim_managementapi:
    hostname: demo-managementapi
    image: graviteeio/management-api:1.24.0
    environment:
      - gravitee_management_mongodb_uri=mongodb://mongodb:27017/gravitee?serverSelectionTimeoutMS=5000&connectTimeoutMS=5000&socketTimeoutMS=5000
      - gravitee_analytics_elasticsearch_endpoints_0=http://elasticsearch:9200
    labels:
      - "traefik.frontend.rule=Host:ac45devapp01.cc.columbia.edu;PathPrefix:/management"
      - "traefik.backend=graviteeio-apim-management-api"
      - "traefik.port=8083"
      - "traefik.docker.network=traefik"
    networks:
      - storage
      - frontend
      - traefik

  demoapp:
    image: myapp:latest
    hostname: demoapp
    # set various environment variables as needed:
    environment:
      - DJANGO_DEBUG=false
    # expose the port just so we can compare direct vs. gateway access
    ports:
      - "9123:9123"
    labels:
      - "traefik.enable=false"
    networks:
      - api
    volumes:
      - local_staticfiles:/var/www/html
    depends_on:
      - nginx

  # If I try to mount local_staticfiles at /usr/share/nginx/html, nginx clobbers my files, so
  # mount at .../static and then symlink
  nginx:
    image: nginx:latest
    hostname: webserver
    volumes:
      - local_staticfiles:/usr/share/nginx/html/static
    labels:
      # look for /openapi/* and /oauth2-redirect.html
      - "traefik.frontend.rule=Host:localhost;PathPrefix:/static/,/openapi/,/oauth2-redirect.html"
      - "traefik.port=80"
    ports:
      - "9090:80"
    networks:
      - frontend
    command: /bin/bash -c "cd /usr/share/nginx/html && ln -sf static/* . && exec nginx -g 'daemon off;'"

```

**Comment by n2ygk** [comment]
PS: here's the traefik version:
```
# docker run -it --rm traefik version
Version:      v1.7.10
Codename:     maroilles
Go version:   go1.11.6
Built:        2019-03-29_12:20:34PM
OS/Arch:      linux/amd64
```

**Comment by n2ygk** [comment]
I restarted things and now traefik is not even routing the /static path, returning a 404. WTF?

See [traefik.log](https://github.com/containous/traefik/files/3122522/traefik.log) for debug output. It looks like the "Configuration received from provider docker" is correct....

**Comment by n2ygk** [comment]
With apologies, all the above is garbage. My config had the wrong hostname (which I coped from my localhost testing). I was getting text/plain because it was a 404 response.

**Comment by Gr33nbl00d** [comment]
I see i really have to put an example together :) It happend with an old
tomcat version. The idea is quite simple the response from server simply
does not contain a content-type at all. I try to make an example

Am Fr., 26. Apr. 2019 um 20:29 Uhr schrieb Alan Crosswell <
notifications@github.com>:

**Comment by thtas** [comment]
I've had this happen using Traefik combined with Geoserver running on Tomcat 8

https://www.github.com/kartoza/docker-geoserver

My fix was to add a mime mapping for css files in to the web.xml file for the app so that css files served up by Tomcat receive a css header.

```xml
<web-app>
   ...
   <mime-mapping>
        <extension>css</extension>
        <mime-type>text/css</mime-type>
    </mime-mapping>
</web-app>
```

Connecting directly to tomcat, the browser (firefox) will interpret CSS files correctly when there is no response header as it just assumes they are css, but when Traefik adds `text/plain firefox won't render the css.

**Comment by Ullaakut** [comment]
Just an update on this from @juliens: the issue seems to be due to [the `net/http` package](https://github.com/golang/go/blob/d016330241c526f09b5ec70cdf17bfd0c257006b/src/net/http/server.go#L1379).

**Comment by traefiker** [comment]
Hi! I'm Træfiker :robot: the bot in charge of tidying up the issues.

I have to close this one because of its lack of activity :disappointed:

Feel free to re-open it or join our [Slack workspace](https://traefik.herokuapp.com/) for more [community #support](https://traefik.slack.com/messages/C0CDT22PJ/).

**Comment by billimek** [comment]
Please don’t close this 🙁

**Comment by dduportal** [comment]
Hi @billimek , could you please provide a reproduction case on this one? I'm not saying this is a bug, but our own @Ullaakut was not able to reproduce the issue back then, neither do I.

If you provide us a `docker-compose.yml` (with Traefik and a backend), or Kubernetes YAMLs, to reproduce this issue, we'll happilly re-open this issue and triage it as expected :)

**Comment by traefiker** [comment]
Closed by #6097.
